packages/web-client/app/components/card-pay/prepaid-card/index.css

```diff
@@ -129,10 +137,19 @@
   right: 0;
   left: 0;
   border-radius: var(--boxel-border-radius) var(--boxel-border-radius) 0 0;
-  height: 102px;
   overflow: hidden;
-  background: var(--header-theme), var(--header-background);
-  background-size: cover;
-  background-position: center;
   z-index: 0;
+  height: 14.375rem;
+}
+
+.prepaid-card__background--color {
+  background: var(--header-background);
+}
+
+.prepaid-card__background--pattern {
+  background-color: var(--header-pattern-color);
+  -webkit-mask-size: contain;
+  -webkit-mask-image: var(--header-theme);
+  mask-size: contain;
+  mask-image: var(--header-theme);
```

Comment:
Has there been discussion of any kind of preprocessing or the like so both the prefixed and non-prefixed versions don’t need to be present? Or is there a guideline on when to use both?

I’m not sure but I suspect some tweaking here would help as the header background is poking out behind the bottom corners of the card for me:

![image](https://user-images.githubusercontent.com/43280/122080857-39169f00-cdc4-11eb-978a-e7a19f9230f0.png)

Maybe the border is desired though?

![image](https://user-images.githubusercontent.com/43280/122081050-606d6c00-cdc4-11eb-9411-5804af9bdffd.png)

I tried a few browsers and they all had the poking 😯

Reply:
Ah! Thanks. I didn't spot the bottom poking out earlier. Will fix this. 

I don't think we've discussed prefixing, though I think it would be helpful. The reason the prefixing is present for this one is because `mask-image` is non-standard and only supported in Firefox, we aim to support Chrome, Safari, and Firefox.